daemons/gptp/windows/daemon_cl/windows_hal.hpp

```diff
@@ -669,12 +695,12 @@ class WindowsTimestamper : public HWTimestamper {
         if(( result = readOID( OID_INTEL_GET_SYSTIM, buf, sizeof(buf), &returned )) != ERROR_SUCCESS ) return false;
 
         now_net = (((uint64_t)buf[1]) << 32) | buf[0];
-        now_net = scaleNativeClockToNanoseconds( now_net );
+		now_net = scaleNativeClockToNanoseconds( now_net );
         *device_time = nanoseconds64ToTimestamp( now_net );
```

Comment:
This code is kind of a mess in terms of formatting. I will add another patch to the series that fixes the formatting only.